microprofile-opentracing/README.adoc

```diff
@@ -36,6 +36,12 @@ include::../shared-doc/use-of-jboss-home-name.adoc[leveloffset=+1]
 // Start the {productName} Standalone Server
 include::../shared-doc/start-the-standalone-server.adoc[leveloffset=+1]
 
+NOTE: Please note the additional parameters defined for the server startup script.
```

Comment:
just drop the "NOTE:", agree the alignment looks weird, your text already has "Please note" anyway :-)

Reply:
@emmartins , oh, you are right :) I've just updated, looks much better now. Ready for re-review :)